clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClient.java

```diff
@@ -113,6 +113,12 @@ public Node leastLoadedNode() {
         }
     }
 
+    public boolean hasReadyNodes() {
+        synchronized (this) {
```

Comment:
@ijuma Thank you for the review. It was a copy-and-paste from the method above. Made both methods synchronized.